### DISCUSSION TITLE
guide/drivers/googledrive

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#55] [Announcements]
# guide/drivers/googledrive

Documentation for alist v3

https://alist.nn.ci/guide/drivers/googledrive.html

**Comment by zhuangyin8** [comment]
Access blocked: rclone’s request is invalid

You can’t sign in because rclone sent an invalid request. You can try again later, or contact the developer about this issue. Learn more about this error
If you are a developer of rclone, see error details.
Error 400: invalid_request

**Reply by SingingDalong** [reply]
Me too...

**Reply by Jue-Xu** [reply]
Same

**Reply by jiawen1929** [reply]
same

**Reply by 2710128764** [reply]
我也是

**Comment by itsmo-ks** [comment]
Getting this error when trying to delete a file or a folder from google drive.

"The user does not have sufficient permissions for this file.: [{global insufficientFilePermissions The user does not have sufficient permissions for this file. }]"

Any solution ?

**Reply by SingingDalong** [reply]
> Getting this error when trying to delete a file or a folder from google drive.
> 
> "The user does not have sufficient permissions for this file.: [{global insufficientFilePermissions The user does not have sufficient permissions for this file. }]"
> 
> Any solution ?

https://singingdalong.blogspot.com/2023/02/Add-Google-Shared-Drive-to-Alist.html

**Reply by itsmo-ks** [reply]
I already mounted it , it's working great, but I can't delete files inside AList , I need to go to google drive and delete it there

**Reply by SingingDalong** [reply]
> I already mounted it , it's working great, but I can't delete files inside AList , I need to go to google drive and delete it there

Get a client_id client_secret from Google Cloud Platform and use it in rclone.
https://singingdalong.blogspot.com/2023/03/Add-Google-Shared-Drive-to-Alist.html

**Reply by itsmo-ks** [reply]
I need to use Rclone , so the one provided in the guide website won't work ?
Because i generated a client and secret id , from my account and used it with this https://alist.nn.ci/tool/google/request to get refresh token

**Reply by SingingDalong** [reply]
> I already mounted it , it's working great, but I can't delete files inside AList , I need to go to google drive and delete it there

Oh, I spelled it wrong. Get a client_id client_secret from Google Cloud Platform and use it for alist.

**Reply by itsmo-ks** [reply]
That's what I did , but I can't delete files

**Reply by SingingDalong** [reply]
> That's what I did , but I can't delete files

Try the lower version of Alist. Occasionally newer versions may introduce bugs.
